wagtailio/static/sass/components/_comparison-table.scss

```diff
@@ -0,0 +1,42 @@
+.comparison-table {
+    align-items: start;
```

Comment:
question: was this added for text alignment? I usually see it associated with `display: flex` so i'm curious what effect it was with that property added

Reply:
It is, I'm not sure why I left this, my bad.